docs/processes/release/tasks/updating-measure-unit.md

```diff
@@ -0,0 +1,128 @@
+* Look for line containing:
+
+  `// End generated MeasureUnit constants`
+* Replace all the generated code in between with the contents of the clipboard
+* Run the MeasureUnitTest.java to ensure that the new code is backward
+  compatible. These compatibility tests are called something like
+  `TestCompatible65`, which tests backward compatibility with ICU 65.
+* Create a compatibility test for ICU 68. Change `testZZZ` to run
+  `generateBackwardCompatibilityTest(“68”)`
+* Run tests.
+* Copy generated test (see instructions above) into MeasureUnitTest.java
+* Run tests again to ensure that new code is backward compatible with itself
+
+## Update ICU4C
+
+* checkout ICU4C
+
+### Update measunit.h
+
+* Change testZZZ to run `generateCXXHConstants(“68”); // ICU 68`.
+* Run MeasureUnitTest.java, copy the generated code (see instructions above).
+* Open $GIT_ROOT/icu4c/source/i18n/unicode/measunit.h. Look for line containing:
+
+  `// Start generated createXXX methods`
+* Look for line:
+
+  `// End generated createXXX methods`
+* Replace all the generated code in between with the contents of the clipboard
+
+### Update measunit.cpp
+
+* Change testZZZ to run generateCXXConstants();
+* Run MeasureUnitTest.java, copy the generated code (see instructions above).
+* Open $GIT_ROOT/icu4c/source/i18n/measunit.cpp. Look for line containing:
+
+  `// Start generated code for measunit.cpp`
+* Look for lines
+
+  `// End generated code for measunit.cpp`
+* Replace all the generated code in between with the contents of the clipboard
+
+### Run C++ tests
+
+* Run `./intltest format/MeasureFormatTest` from `test/intltest` to ensure new
+  code is backward compatible.
+* Create a compatibility test for ICU 68. Change `testZZZ` in eclipse to run
+  `generateCXXBackwardCompatibilityTest(“68”)`
+* Run tests.
+* Copy generated test (see instructions above) into
+  $GIT_ROOT/icu4c/source/test/intltest/measfmttest.cpp. Make other necessary
+  changes to make test compile. You can find these changes by searching for
+  `TestCompatible65()`
+* Run tests again to ensure that new code is backward compatible with itself
+
+## Finishing changes
+
+These last changes are necessary to permanently record the ICU version number of
+any new measure units. Without these changes any new functions for this release
+will be considered new for the next release too.
+
+* Change `testZZZ` to run `updateJAVAVersions(“68”);`
+* Run MeasureUnitTest.java, copy the generated code (see instructions above).
+* Append the clipboard contents to the values of the JAVA_VERSIONS variable
+  near the top of MeasureUnitTest.java.
+
+  **Important:** what you are copying are just the new functions for the current
+  ICU version, in this case 68. Therefore append, do not replace.
+
+## Updating units.txt and unitConstants
+
+The standard ldml2icu process is used to update ICU's resource files (see
+[cldr-icu-readme.txt](../../../../icu4c/source/data/cldr-icu-readme.txt)).
```

Comment:
I don't yet know whether absolute links work on GitHub. I'm going for relative links anyway: because for local access, an editor might not know where the git repository root is, whereas relative paths are always at least a known concept.

Reply:
Relative makes sense. Marking as resolved to reduce noise.